pkgs/servers/uwsgi/default.nix

```diff
@@ -59,20 +60,21 @@ in
 
 stdenv.mkDerivation rec {
   pname = "uwsgi";
-  version = "2.0.20";
+  version = "2.0.21";
 
-  src = fetchurl {
```

Comment:
You forgot to remove fetchurl from line 1

Reply:
indeed. fixed